dynatrace/tests/samples/host_response.json

```diff
@@ -0,0 +1,107 @@
+[
+    {
+        "entityId": "HOST-6AAE0F78BCF2E0F4",
+        "displayName": "EX01.stackstate.lab",
+        "discoveredName": "EX01.stackstate.lab",
+        "firstSeenTimestamp": 1600180637597,
+        "lastSeenTimestamp": 1601447916389,
+        "tags": [],
+        "fromRelationships": {},
+        "toRelationships": {
+            "isProcessOf": [
+                "PROCESS_GROUP_INSTANCE-0EC2543319EDF0A3"
+            ],
+            "isSiteOf": [
+                "GEOLOC_SITE-C9B2D39AD8A7FA9A",
+                "AWS_AVAILABILITY_ZONE-BA734CACA276B570"
+            ],
+            "runsOn": [
+                "PROCESS_GROUP-55BB99EBACB51463"
+            ]
+        },
+        "osType": "WINDOWS",
+        "osArchitecture": "X86",
+        "osVersion": "Windows Server 2016 Datacenter 1607, ver. 10.0.14393",
+        "hypervisorType": "XEN",
+        "ipAddresses": [
+            "3.249.27.126",
+            "172.31.29.43"
+        ],
+        "bitness": "64bit",
+        "cpuCores": 2,
+        "logicalCpuCores": 2,
+        "cloudType": "EC2",
+        "monitoringMode": "FULL_STACK",
+        "networkZoneId": "default",
+        "agentVersion": {
+            "major": 1,
+            "minor": 201,
+            "revision": 127,
+            "timestamp": "20200925-143440",
+            "sourceRevision": ""
+        },
+        "consumedHostUnits": 0.5,
```

Comment:
@lowdewijk  correct this is coming from datadog code where we expect only `string, int, dictionary, list or None value` for `data` section and that is why I have to convert this value.

Reply:
Fix in the base class. StackState can handle these types just perfectly.

Reply:
We have the ticket to implement this scenario in the core agent this sprint because the agent doesn't handle this type and till then we have it converted as a string.
@zandrewitte @lowdewijk @TizianoPerrucci

Reply:
Ok, then please a put a note in that ticket to fix it in this place right after.